src/quantities.jl

```diff
@@ -57,9 +57,15 @@ end
 //(x::AbstractQuantity, y::Complex) = Quantity(//(x.val, y), unit(x))
 
 for f in (:div, :fld, :cld)
-    @eval function ($f)(x::AbstractQuantity, y::AbstractQuantity)
-        z = uconvert(unit(y), x)        # TODO: use promote?
-        ($f)(z.val,y.val)
+    @eval begin
+        function ($f)(x::AbstractQuantity, y::AbstractQuantity)
+            z = uconvert(unit(y), x)        # TODO: use promote?
+            ($f)(z.val,y.val)
+        end
+
+        ($f)(x::Number, y::AbstractQuantity) = ($f)(x, ustrip(y)) / unit(y)
+
+        ($f)(x::AbstractQuantity, y::Number) = unit(x) * ($f)(ustrip(x), y)
```

Comment:
I'd follow that division does here:
```suggestion
        ($f)(x::Number, y::AbstractQuantity) = Quantity(($f)(x, ustrip(y)), unit(x) / unit(y))

        ($f)(x::AbstractQuantity, y::Number) = Quantity(($f)(ustrip(x), y), unit(x) / unit(y))
```